### ISSUE TITLE
Add register/login functionality

### ISSUE THREAD

**Issue by bstratton283** [bwwagner/ProjectSnowCrash#10]
Need to associate the user with their checking/savings accounts and redirect the user back to a login page if they try to access a page that requires login

**Comment by tcspal5892** [comment]
Ok, Brad and I worked on this for a few hours last night.  We made good progress and I pushed those changes up this afternoon.  Still need to do a couple things.  Also, in order to make this work on your machine, you will need to install a couple nuget packages after syncing your project up.  I will give you the commands below. 

Install-Package Microsoft.AspNet.Identity.Core
Install-Package Microsoft.AspNet.Identity.EntityFramework
Install-Package Microsoft.AspNet.Identity.Owin
Install-Package Microsoft.Owin.Host.SystemWeb

Open the package manager console and paste those in.  Just follow what prompts come up and let them do what they will.  If anyone has questions ask Brad or myself.
